test/libsolidity/semanticTests/functionCall/failed_create.sol

```diff
@@ -18,17 +18,17 @@ contract C {
 // compileViaYul: also
 // ----
 // constructor(), 20 wei
-// gas irOptimized: 219233
-// gas legacy: 288299
-// gas legacyOptimized: 177933
+// gas irOptimized: 212915
+// gas legacy: 294569
+// gas legacyOptimized: 174699
```

Comment:
I see that the gas here is still at `294569` but it's no longer a change. Was part of this extracted into a separate PR that already went in?

Reply:
Ah, no, it's still a change. I was looking at some partial diff rather than the whole thing.

So anyway, do we have an explanation for why this happens even though the PR only removes code?